pulsar-broker/src/test/java/com/yahoo/pulsar/broker/loadbalance/SimpleLoadManagerImplTest.java

```diff
@@ -441,8 +436,7 @@ public void testNamespaceBundleStats() {
 
     @Test
     public void testBrokerHostUsage() {
-        when(pulsar1.getConfiguration().getLoadBalancerHostUsageScriptPath()).thenReturn("usageScript");
-        BrokerHostUsage brokerUsage = new BrokerHostUsage(pulsar1);
+        BrokerHostUsage brokerUsage = new LinuxBrokerHostUsageImpl(pulsar1);
```

Comment:
This should either fallback to generic host usage impl or disable the test on non-linux.